build-tools/packages/version-tools/src/index.ts

```diff
@@ -40,5 +41,8 @@ export {
 	VersionScheme,
 } from "./schemes";
 export { bumpRange, detectBumpType, isPrereleaseVersion, getPreviousVersions } from "./semver";
-export { getIsLatest, getSimpleVersion } from "./versions";
+export {
+	getIsLatest,
+	getSimpleVersion,
+} from "./versions";
```

Comment:
No difference except formatting - formatter is unstable?

Reply:
manually fixed the formatting :)